.gitignore

```diff
@@ -2,3 +2,10 @@ node_modules/*
 .tmp/
 npm-debug*
 package-lock.json
+.DS_Store
+
+test/migrations/generated/migrations/*
+!test/migrations/generated/migrations/.gitkeep
```

Comment:
Why ignore gitkeeps?

Reply:
This is the secret to ignoring everything except .gitkeeps

Reply:
Ah, thanks!  Sorry, missed that.